developer-docs-site/src/contributors.json

```diff
@@ -465,6 +561,26 @@
       "name": "David Wolinsky",
       "email": "isaac.wolinsky@gmail.com",
       "username": "davidiw"
+    },
+    {
+      "name": "0xbe1",
+      "email": "101405096+0xbe1@users.noreply.github.com",
+      "username": "0xbe1"
+    },
+    {
+      "name": "Aaron",
+      "email": "lightmark@gmail.com",
+      "username": null
```

Comment:
How are you generating this btw?  We can fix some of these people's usernames

Reply:
If the commit email isn't a username@users.noreply.github.com address, then the username is retrieved from this list: https://github.com/aptos-labs/aptos-core/blob/e6bc2844a9909dd7ae332dda4c54b1e3b8fd7d38/developer-docs-site/scripts/contributors.js#L7-L9
We would need to add an entry to this list to resolve this email correctly.